drivers/net/bnxt/bnxt_ethdev.c

```diff
@@ -1333,7 +1333,9 @@ bnxt_dev_init(struct rte_eth_dev *eth_dev)
 	}
 
 	/* Forward all requests if firmware is new enough */
-	if (1)
+	if (((bp->fw_ver >= ((20 << 24) | (6 << 16) | (100 << 8))) &&
+	    (bp->fw_ver < ((20 << 24) | (7 << 16)))) ||
+	    ((bp->fw_ver >= ((20 << 24) | (8 << 16)))))
```

Comment:
We don't need to check for the minor version in 20.8 ? Or because by the time it comes to production, the change will be available anyway?